### ISSUE TITLE
newline-after-var declare and export

### ISSUE THREAD

**Issue by corbinu** [eslint/eslint#2325]
Another issue with newline-after-var when it is enabled I get an error when exporting one let after another. This is in version v0.19.0
```
export let A = "A";
export let B = "B";
```
Which seems to be an error given that this does not 
```
 let A = "A";
 let B = "B";
```

the message is `warning  Expected blank line after variable declarations  newline-after-var`

**Comment by nzakas** [comment]
The second example definitely should warn.

**Comment by gyandeeps** [comment]
I think the second example does give out errors.

**Comment by nzakas** [comment]
I would leave import out of this rule.

**Comment by btmills** [comment]
newline-after-var has an exception for consecutive variable declarations so that only the last one is followed by a blank line. The second example is behaving as expected.

**Comment by nzakas** [comment]
Ah, thanks for the clarification.
